packages/components/src/Components/TagCount/TagCount.js

```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import './tagCount.scss';
 
-const TagCount = ({count, onTagClick, ...props}) => {
+const TagCount = ({count, onTagClick, className, ...props}) => {
     return (
-        <div className="ins-c-tag-count" onClick={onTagClick}>
+        <div {...props} className={classNames("ins-c-tag-count", className)} onClick={onTagClick}>
```

Comment:
Is there a reason you're using the `classNames` import here? seems like you could just do 
```
className=`ins-c-tag-count  ${className}`
```

Reply:
That's just how @karelhala suggested. I don't know if there is a functional difference

Reply:
Note the `s` in classNames

We definitely want to spread the user `className` prop. However, I don't think we need to use the import `classNames` to accomplish that.

Reply:
Seems like it works fine how you're suggesting so let's go with that